docs/coordinates/satellites.rst

```diff
@@ -0,0 +1,87 @@
+.. include:: references.txt
+
+.. _astropy-coordinates-satellites:
+
+Working with Earth satellites using Astropy Coordinates
+*******************************************************
+
+Finding ``TEME`` coordinates from Two-Line-Ephemerides (TLE)
+============================================================
+
+There is currently no support in `astropy.coordinates` for computing satellite orbits
+from the TLE orbital elements avaiable from a source like `Celestrak <http://celestrak.com/>`_.
+Full support for handling TLE files is available in the `Skyfield <http://rhodesmill.org/skyfield/>`_
+library, but if you do not wish to install that library, advice for dealing with satellite data in
+astropy is below.
```

Comment:
```suggestion
library, but some advice for dealing with satellite data in astropy is below.
```
Suggestion: slightly more neutral phrasing to not make it seem like we're saying there's a concrete reason not to install skyfield.